docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
       - db:/var/lib/mysql
 
   concerto:
-    image: concerto
+    image: concertong
```

Comment:
Why have you changed this? If you like to change the name, you shall update the README.md

Reply:
good catch @jo-gross, that's not even a public image. I've reverted it in pr #1576